src/lib/rust.ts

```diff
@@ -300,13 +305,36 @@ export const ensureCargoDependency = async ({
     });
 
   try {
-    if (process.env.CI === "true") {
+    if (
+      force ||
+      ![MARINE_CARGO_DEPENDENCY, MREPL_CARGO_DEPENDENCY].includes(name)
+    ) {
+      throw new Error("Install using cargo");
+    }
+
+    if (["darwin", "linux"].includes(platform()) && arch() === "x64") {
```

Comment:
Arch on my laptop is arm64, yet I'm able to use x86_64 binaries. Maybe check if `platform = darwin && arch = arm64` and download x86 anyway?

Reply:
it will not break anything because it will install from cargo like before. Anatoly doesn't build for arm64. I am not sure if it works good with rosetta. Can you please download x86_64 binary on your mac and check if it works?